R/zchunk_L223.electricity_USA.R

```diff
@@ -1,58 +1,68 @@
+#' \code{L223.StubTechFixOut_elec_USA}, \code{L223.StubTechFixOut_hydro_USA}, \code{L223.StubTechProd_elec_USA},
+#' \code{L223.StubTechMarket_elec_USA}, \code{L223.StubTechMarket_backup_USA}, \code{L223.StubTechElecMarket_backup_USA},
+#' \code{L223.StubTechCapFactor_elec_wind_USA}, \code{L223.StubTechCapFactor_elec_solar_USA}. The corresponding file in the
 #' original data system was \code{L223.electricity_USA.R} (gcam-usa level2).
-#' @details Describe in detail what this chunk does.
+#' @details This chunk disaggregtes the USA electricity sector to subnational grid regions and states.
 #' @importFrom assertthat assert_that
 #' @importFrom dplyr filter mutate select
 #' @importFrom tidyr gather spread
-#' @author YourInitials CurrentMonthName 2017
-#' @export
-module_gcam.usa_L223.electricity_USA_DISABLED <- function(command, ...) {
+#' @author RC Oct 2017
+module_gcam.usa_L223.electricity_USA <- function(command, ...) {
   if(command == driver.DECLARE_INPUTS) {
     return(c(FILE = "gcam-usa/states_subregions",
              FILE = "energy/calibrated_techs",
              FILE = "gcam-usa/NREL_us_re_technical_potential",
              FILE = "energy/A23.globaltech_eff",
-             "L114.CapacityFactor_wind_state",
+             FILE = "temp-data-inject/L114.CapacityFactor_wind_state",
              "L119.CapFacScaler_PV_state",
              "L119.CapFacScaler_CSP_state",
-             "L223.Supplysector_elec",
-             "L223.ElecReserve",
-             "L223.SubsectorLogit_elec",
-             "L223.SubsectorShrwt_elec",
-             "L223.SubsectorShrwtFllt_elec",
-             "L223.SubsectorShrwt_nuc",
-             "L223.SubsectorShrwt_renew",
-             "L223.SubsectorInterp_elec",
-             "L223.SubsectorInterpTo_elec",
-             "L223.StubTech_elec",
-             "L223.StubTechEff_elec",
-             "L223.StubTechCapFactor_elec",
-             "L223.GlobalIntTechBackup_elec",
+             FILE = "temp-data-inject/L223.Supplysector_elec",
+             FILE = "temp-data-inject/L223.ElecReserve",
+             FILE = "temp-data-inject/L223.SubsectorLogit_elec",
+             FILE = "temp-data-inject/L223.SubsectorShrwtFllt_elec",
+             FILE = "temp-data-inject/L223.SubsectorShrwt_nuc",
+             FILE = "temp-data-inject/L223.SubsectorShrwt_renew",
+             FILE = "temp-data-inject/L223.SubsectorInterp_elec",
+             FILE = "temp-data-inject/L223.SubsectorInterpTo_elec",
+             FILE = "temp-data-inject/L223.StubTech_elec",
+             FILE = "temp-data-inject/L223.StubTechEff_elec",
+             FILE = "temp-data-inject/L223.StubTechCapFactor_elec",
+             FILE = "temp-data-inject/L223.GlobalIntTechBackup_elec",
              "L1231.in_EJ_state_elec_F_tech",
              "L1231.out_EJ_state_elec_F_tech",
-             "L1232.out_EJ_sR_elec",
-             "L126.out_EJ_state_td_elec"))
+             "L1232.out_EJ_sR_elec"))
```

Comment:
This file is only used as a precursor for a conditional output `L223.Production_USAelec`, which is not produced currently. So running the driver gets the warning message: `Inputs L1232.out_EJ_sR_elec don't appear as precursors for any outputs`.

Reply:
See comment below--let's eliminate the warning message.